wikitongues/wikitongues/data_store/airtable/airtable_language_data_store.py

```diff
@@ -93,8 +94,12 @@ def list_languages(self, page_size=100, max_records=None):
 
         result = ErrorResponse()
 
+        if kwargs.get('offset') is not None:
```

Comment:
A couple notes on this:
* The offset value actually comes from the response object (i.e. `response` on line 101). You query the Airtable API to list the records, and you get back a list containing `max_records` records, with an offset value. Then you can query the API again, passing the offset as a parameter, and you'll get a list starting where the previous query left off. (So if the first list contained records 1-100, the second will contain 101-200.)
  * Therefore we want to write the file after receiving the API response on line 101
  * This is better explained in the [Airtable API documentation](https://airtable.com/api)
    * You may need to open a (free) Airtable account to access the documentation. It is interactive, and shows you sample queries and responses based on the tables in your account, and it is very helpful
* Let's move the code that writes the offset file into a new function called `write_offset`, and that function will go in the file you created named offset_reader.py. 
  * Then rename offset_reader.py to something like offset_utility.py
  * The `read_offset` and `write_offset` functions can both be [static](https://realpython.com/instance-class-and-static-methods-demystified/)

Reply:
Thanks for the tips! I was working on it and I'm not quite sure how to call the offset value from the response object? Also when testing, should the MockResponse contain an offset value?

Reply:
After we load in the json response object [here](https://github.com/wikitongues/Language-Indexing/blob/2b95d9aa66324d37d59c3578d52e05fa81acffa3/wikitongues/wikitongues/data_store/airtable/airtable_language_data_store.py#L108), you should be able to do something like `json_obj.get('offset')` to get the offset value. 

And yes, updating the unit tests and mocks would be excellent!